packages/kbn-optimizer/limits.yml

```diff
@@ -105,7 +105,7 @@ pageLoadAssetSize:
   screenshotting: 22870
   searchprofiler: 67080
   security: 65433
-  securitySolution: 273763
+  securitySolution: 339077
```

Comment:
Will be addressed in a follow-up PR. https://github.com/elastic/kibana/issues/143532